### PR TITLE
Release 2.45.3

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

**Review by Nexucis — APPROVED** [review]

**Review by bboreham — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.45.3 / 2024-01-24
+
+This release contains security fixes in dependencies and has been built with go1.21.6. #13450.
+
+* [BUGFIX] TSDB: Remove double memory snapshot on shutdown. #13450
```
Comment (juliusv) [inline CHANGELOG.md:7]: Shouldn't this be a different PR number than above (which is this PR itself)?

Comment (roidelapluie) [reply CHANGELOG.md:7]: ack
